abce/trade.pyx

```diff
@@ -659,7 +659,7 @@ class Trade:
             self._reject(offer)
         self._polled_offers = {}
 
-    cdef _reject(self, Offer offer):
+    def _reject(self, Offer offer):
```

Comment:
I have to rename this to `def` because otherwise cython 0.27.3 won't compile the trade.pyx file.

Reply:
I can not reproduce this problem and it comes at a mayor speed penalty. We should investigate this better.

Reply:
Yeah removed.